app/controllers/posts_controller.rb

```diff
@@ -29,4 +29,4 @@ def post_params
   end
 
   private :post_params
-end
```

Comment:
* As mentioned in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Make%20sure%20that%20the%20N%2B1%20problem%20is%20solved%20when%20fetching%20all%20posts%20and%20their%20comments%20for%20a%20user.) Kindly ensure that the N+1 problem is solved when fetching all posts and their comments for a user. You need to solve the n+1 issue in your code , HINT 💭 you can fetch all posts and comments belonging to a user by using this code `@posts = @user.posts.includes(:comments)`

Reply:
@Davidosky007 
thanks for your feedback,
your suggestion applied.